docs clarification - more details about activating virtual environment? `make requirements` fails?

on an Apple M2 Pro, macOS 13.4.1 (c) 

after `make create_environment` the log reads
```
>>> New virtualenv created. Activate with:
workon street-view-green-view
```
but that command errors with `command not found: workon`
progressing to step 2 failed until I ran `source venv/bin/activate` which is not mentioned in the docs
even after activating the virtual environment, running `make requirements` failed with
```
/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/Resources/Python.app/Contents/MacOS/
Python: can't open file '/Users/danbjoseph/GitHub/americanredcross/street-view-green-view/test_environment.py': 
[Errno 2] No such file or directory
make: *** [test_environment] Error 2
```
however, running `pip install .` did work

<img width="971" alt="Screenshot 2024-03-02 at 10 47 28 AM" src="https://github.com/AmericanRedCross/street-view-green-view/assets/4806884/3db2c5a4-e58c-4e72-bc31-dd93339258ef">

--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,7 @@ endif
 #################################################################################
 
 ## Install Python Dependencies
-requirements: test_environment
+requirements: 
 	$(PYTHON_INTERPRETER) -m pip install -U pip setuptools wheel
 	$(PYTHON_INTERPRETER) -m pip install -r requirements.txt
 
@@ -59,10 +59,6 @@ else
 	@bash -c "source `which virtualenvwrapper.sh`;mkvirtualenv $(PROJECT_NAME) --python=$(PYTHON_INTERPRETER)"
 	@echo ">>> New virtualenv created. Activate with:\nworkon $(PROJECT_NAME)"
 endif
-
-## Test python environment is setup correctly
-test_environment:
-	$(PYTHON_INTERPRETER) test_environment.py
 
 #################################################################################
 # PROJECT RULES                                                                 #

--- a/Makefile
+++ b/Makefile
@@ -53,11 +53,8 @@ else
 endif
 		@echo ">>> New conda env created. Activate with:\nsource activate $(PROJECT_NAME)"
 else
-	$(PYTHON_INTERPRETER) -m pip install -q virtualenv virtualenvwrapper
-	@echo ">>> Installing virtualenvwrapper if not already installed.\nMake sure the following lines are in shell startup file\n\
-	export WORKON_HOME=$$HOME/.virtualenvs\nexport PROJECT_HOME=$$HOME/Devel\nsource /usr/local/bin/virtualenvwrapper.sh\n"
-	@bash -c "source `which virtualenvwrapper.sh`;mkvirtualenv $(PROJECT_NAME) --python=$(PYTHON_INTERPRETER)"
-	@echo ">>> New virtualenv created. Activate with:\nworkon $(PROJECT_NAME)"
+	$(PYTHON_INTERPRETER) -m venv .venv
+	@echo ">>> New virtual environment created. Activate with:\nsource .venv/bin/activate"
 endif
 
 #################################################################################

--- a/README.md
+++ b/README.md
@@ -36,6 +36,7 @@ If you are interested in joining the project, please check out [`CONTRIBUTING.md
 
 1. Create a Python virtual environment.
      - You can use the shortcut command `make create_environment`.
+     - The log will tell you how to activate the environment. Do so with: `source .venv/bin/activate`
 2. Install requirements.
     ```bash
     pip install -r requirements.txt

